src/Playwright/Transport/StdIOTransport.cs

```diff
@@ -54,7 +55,7 @@ internal StdIOTransport()
         };
         _process.BeginErrorReadLine();
 
-        ScheduleTransportTask(GetResponseAsync, _readerCancellationSource.Token);
+        _getResponseTask = ScheduleTransportTaskAsync(GetResponseAsync, _readerCancellationSource.Token);
```

Comment:
Can we add a test for this, was it actually fixing something? How did you find out about this?

Reply:
After upgrade to newer analyzers there are existing issues detected, like this:
https://github.com/microsoft/playwright-dotnet/actions/runs/5114152936/jobs/9194105803?pr=2556#step:8:11